lib/segment/benches/dynamic_mmap_flags.rs

```diff
@@ -17,7 +17,7 @@ fn dynamic_mmap_flag_count(c: &mut Criterion) {
     let stopped = AtomicBool::new(false);
 
     // Build dynamic mmap flags with random deletions
-    let mut dynamic_flags = DynamicMmapFlags::open(dir.path()).unwrap();
+    let mut dynamic_flags = DynamicMmapFlags::open(dir.path(), false).unwrap();
```

Comment:
Not a big fan of the boolean argument TBH
Makes code less readable.

Reply:
Removed for now :)